modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -274,17 +274,13 @@ By default, database metrics include:
 
 [options="header",cols="<3m,<4"]
 |===
-|Name
-|Description
+|Name |Description
 
-|<prefix>.dbms.routing.query.count.local
-|The total number of queries executed locally. (counter)
+|<prefix>.dbms.routing.query.count.local|The total number of queries executed locally. (counter)
 
-|<prefix>.dbms.routing.query.count.remote_internal
-|The total number of queries executed remotely to a member of the same cluster. (counter)
+|<prefix>.dbms.routing.query.count.remote_internal|The total number of queries routed over to another member of the same cluster. (counter)
 
-|<prefix>.dbms.routing.query.count.remote_external
-|The total number of queries executed remotely to a member of a different cluster. (counter)
+|<prefix>.dbms.routing.query.count.remote_external|The total number of queries routed over to a server outside the cluster, i.e. DBMS. (counter)
```

Comment:
What do you mean by ` i.e. DBMS.`?

Reply:
let's remove it. I don't think that this metric can be applied to a single server.